packages/lab/src/form-field/FormLabel.tsx

```diff
@@ -25,6 +27,10 @@ export interface FormLabelProps
    * The label value for the FormLabel
    */
   label?: string;
+  /**
+   * Location of label is left; default false
+   */
+  labelLeft?: boolean;
```

Comment:
Not entirely sure whether making the label aware of it's position is a good idea. If this is the only way to do it, probably keep this the similar fashion of the prop from form field.

`labelPlacement="left"`